Assets/MRTK/Core/Utilities/Editor/SpatializerUtilities.cs

```diff
@@ -38,8 +38,8 @@ public static bool CheckSettings()
             // Check to see if an audio spatializer is configured.
             if (string.IsNullOrWhiteSpace(spatializerName))
             {
-                // A spatializer has not been configured.
-                return false;
+                // The user chose to not initialize a spatializer so we are set correctly
```

Comment:
I think the only odd side effect here is that if the spatializers do change (i.e. added/removed) then we won't pop up - i.e. it's odd that if you have one configured it will show up on spatializer count change, but not when it's not selected.

It's almost like that should be moved first then to work around that?